pages.es/common/7z.md

```diff
@@ -30,3 +30,7 @@
 - Lista el contenido de un archivo comprimido:
 
 `7z l {{archivo_comprimido.7z}}`
+
+- Esteblece el nivel de compresión más alto significa más compresión, pero será más lento):
```

Comment:
, (pero será más lento):

Reply:
What do you think about the suggestion below?

Reply:
Est**a**blece el nivel de compresión más alto significa más compresión, pero será más lento: